Scoop/NetworkManager.swift

```diff
@@ -100,13 +100,37 @@ class NetworkManager {
                 }
             }
         }
+
+    func deleteUser(netid: String, first_name: String, last_name: String, grade: String, phone_number: String, pronouns: String, prof_pic: String, prompts: [UserAnswer], completion: @escaping (Result<BaseUser, Error>) -> Void) {
```

Comment:
nit but this violates the line character limit for linting

Reply:
would i put the parameters on separate lines like earlier or just divide it up into more lines

Reply:
```suggestion
    func deleteUser(
        netid: String,
        first_name: String,
        last_name: String,
        grade: String,
        phone_number: String,
        pronouns: String,
        prof_pic: String,
        prompts: [UserAnswer],
        completion: @escaping (Result<BaseUser, Error>) -> Void
    ) {
```
Does it make more sense now?